<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWgt</class>
 <widget class="QWidget" name="LoginWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录 - Eva的商城</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 背景图 + 半透明渐变遮罩（可将 border-image 改为你的大背景图） */
QWidget#LoginWgt {
    border-image: url(:/images/login/bg_login.jpg) 0 0 0 0 stretch stretch;
    font-family: &quot;微软雅黑&quot;;
    color: #5a4c74;
}
/* 可选：如果不想用整图，可注释上面 border-image，改用纯渐变
background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,
            stop:0 #efeafd, stop:0.45 #e0d6f8, stop:1 #c6b4ef);
*/
QLabel { font-family:&quot;微软雅黑&quot;; }
   </string>
  </property>
  <layout class="QHBoxLayout" name="rootLayout">
   <property name="leftMargin">
    <number>28</number>
   </property>
   <property name="topMargin">
    <number>28</number>
   </property>
   <property name="rightMargin">
    <number>28</number>
   </property>
   <property name="bottomMargin">
    <number>28</number>
   </property>
   <item>
    <spacer name="midSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frameLogin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame#frameLogin {
    background: rgba(255,255,255,0.78);
    border-radius: 30px;
    border: 1.8px solid #e2daf6;
    /* backdrop-filter 在某些平台不生效，代码层可用 QGraphicsBlurEffect 实现 */
}
QLabel#lab_title {
    color:#6c5b7b;
    font: 700 22pt &quot;微软雅黑&quot;;
    letter-spacing:2px;
}
QLineEdit {
    font: 12pt &quot;微软雅黑&quot;;
    color:#5a4c74;
    background:#f5f3fa;
    border:1.5px solid #d6c9f7;
    border-radius:14px;
    padding:6px 12px;
}
QLineEdit:focus {
    background:#ffffff;
    border:2px solid #a18cd1;
}
QPushButton {
    font: 13pt &quot;微软雅黑&quot;;
    font-weight:600;
    border-radius:18px;
    padding:8px 20px;
}
QPushButton#btn_login {
    background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,stop:0 #b8a4e6, stop:1 #a18cd1);
    color:#ffffff;
    border:none;
}
QPushButton#btn_login:hover { background:#a18cd1; }
QPushButton#btn_register {
    background:rgba(194,182,243,0.35);
    color:#7a6599;
    border:1.4px solid #bdaee8;
}
QPushButton#btn_register:hover { background:rgba(194,182,243,0.55); }
QToolButton#btn_togglePwd {
    border:none;
    background:transparent;
    padding:0 4px;
    color:#7e6996;
    font: 12pt &quot;微软雅黑&quot;;
}
QToolButton#btn_togglePwd:hover { color:#a18cd1; }
QCheckBox {
    spacing:6px;
    font:11pt &quot;微软雅黑&quot;;
    color:#6a5a84;
}
QCheckBox::indicator {
    width:16px; height:16px;
    border:1px solid #a18cd1;
    border-radius:5px;
    background:#f5f3fa;
}
QCheckBox::indicator:checked {
    background:qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,
               stop:0 #c2b6f3, stop:1 #a18cd1);
    image:none;
}
#lab_forget, #lab_register_link {
    color:#8b78c4;
    font:11pt &quot;微软雅黑&quot;;
}
#lab_forget:hover, #lab_register_link:hover {
    color:#a18cd1;
    text-decoration: underline;
}
      </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="loginLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <widget class="QLabel" name="lab_title">
        <property name="text">
         <string>账号登录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="userRow">
        <item>
         <widget class="QLineEdit" name="line_user">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>用户名 / 邮箱 / 手机号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pwdRow">
        <item>
         <widget class="QLineEdit" name="line_password">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_togglePwd">
          <property name="toolTip">
           <string>显示/隐藏密码</string>
          </property>
          <property name="text">
           <string>👁</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="optionRow">
        <item>
         <widget class="QCheckBox" name="chk_remember">
          <property name="text">
           <string>记住我</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="optSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lab_forget">
          <property name="text">
           <string>忘记密码?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="btnRow">
        <item>
         <widget class="QPushButton" name="btn_login">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>登 录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="btnSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_register">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>注 册</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="lowerSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_register_link">
        <property name="text">
         <string>还没有账号？立即注册 &gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
